app/services/migration/migrators/application.rb

```diff
@@ -1,13 +1,16 @@
 module Migration::Migrators
   class Application < Base
-    ATTRIBUTES_TO_COMPARE = %w[
+    ATTRIBUTES = %w[
+      created_at
```

Comment:
I think in terms of application probably not safe bringing along the timestamps as it is an existing model in NPQ 🤔

Reply:
Is it worth keeping the earliest `created_at` and latest `updated_at`?

Reply:
So Application will almost always have an earlier timestamp as it is created before being created in ECF, in terms of updated at, it will all probably be set to the day after migrators, so not too worried about it, as we're asking providers to sync everything after migration